db/seeds.rb

```diff
@@ -4,14 +4,13 @@
 example_note = "Here is a note that will hopefully demonstrate how somebody might use the notes field to write a long note."
 
 applications = [
-  { name: "Calendars" },
-  { name: "Content store" },
+  { name: "Asset Manager", shortname: "asset-manager" },
+  { name: "Content store", shortname: "content-store" },
   { name: "Feedback" },
   { name: "Frontend", status_notes: example_note },
   { name: "Licensify" },
-  { name: "Places Manager" },
+  { name: "Places Manager", shortname: "places-manager" },
```

Comment:
These shortnames shouldn't be necessary, IIRC? I thought we hyphenate multi-worded application names by default, and only use the `shortname` if we need to override that (a la `smartanswers`)?

Reply:
It seems like the `shortname` is provided in this json block - https://docs.publishing.service.gov.uk/apps.json for each app, this is overriding the default shortname which would have been hyphenated. As this only affects the local dev stack I'll leave the seeds as they are for now and will write up an issue on it.